lib/importers/twilio/saver.rb

```diff
@@ -5,15 +5,27 @@ def initialize(calls:)
         @calls = calls
       end
 
-      def store_valid_by_date
+      def save
+        ActiveRecord::Base.transaction do
+          save_calls!
+          save_valid_by_date!
+        end
+      end
+
+      def save_calls!
+        @calls.each do |call|
+          TwilioCall.find_by(uid: call.uid) ||
```

Comment:
Could this be collapsed into a `find_or_create_by!`? I know you don't do anything with the result but it would be tidier.